lib/command/command.js

```diff
@@ -432,6 +441,17 @@ internals.isCommandAllowed = function (command, slackResponse, slackData) {
   return true;
 };
 
+internals.isAllowedChannel = function (command, slackResponse, slackData) {
+  if (command && command.allowedChannels) {
+      var getAllChannelsByName =  _.map(slackData.channels, 'name');
+      var allowedChannelList= _.intersection(getAllChannelsByName,command.allowedChannels);
+      var currentActiveChannel = _.find(slackData.channels, ['id', slackResponse.channel]);
```

Comment:
I want to keep it consistent with `allowed_user` https://github.com/usubram/slack-quick-bots/blob/master/lib/command/command.js#L423